### ISSUE TITLE
Discord -> Minecraft NPE

### ISSUE THREAD

**Issue by animator013** [chunkaligned/fabric-discord-integration#15]
Hi there,

installed your mod to my fabric server 1.19.4 and encountered this issue when sending message to discord channel. Messages from game are properly sent to discord but the link from discord to game is broken somehow.

Attaching stacktrace:

```
mc_1  | java.lang.NullPointerException: Cannot invoke "net.dv8tion.jda.api.entities.Member.getEffectiveName()" because the return value of "net.dv8tion.jda.api.entities.Guild.getMember(net.dv8tion.jda.api.entities.UserSnowflake)" is null
mc_1  |         at dev.hkva.discord.DiscordIntegrationMod.onDiscordChat(DiscordIntegrationMod.java:268) ~[fabric-discord-integration-1.2.3.jar:?]
mc_1  |         at dev.hkva.discord.callback.DiscordChatCallback.lambda$static$0(DiscordChatCallback.java:12) ~[fabric-discord-integration-1.2.3.jar:?]
mc_1  |         at dev.hkva.discord.DiscordBot.onMessageReceived(DiscordBot.java:79) ~[fabric-discord-integration-1.2.3.jar:?]
mc_1  |         at net.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:424) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:96) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:171) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.handle.MessageCreateHandler.handleInternally(MessageCreateHandler.java:136) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:984) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:870) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:848) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:1023) ~[net_dv8tion_jda-5.0.0-beta.1-77bb473a6ea954a9.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
mc_1  |         at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45) ~[com_neovisionaries_nv-websocket-client-2.14-4bf94d210c4dd2f4.jar:?]
```

**Comment by hkva** [comment]
Hi, I just now got the time to look at this. Somehow the bot isn't able to see the author of the message it's trying to relay from Discord to Minecraft. Could you try enabling the "server members" gateway intent on your bot's developer portal page?

![image](https://github.com/chunkaligned/fabric-discord-integration/assets/91440203/ca040514-6d52-4449-909d-f55d98b9f9e0)

**Comment by animator013** [comment]
Hi there,

I already have that enabled. 

![image](https://github.com/chunkaligned/fabric-discord-integration/assets/4902822/145c5b11-ec7c-499e-9bc1-17442b6ca9aa)

**Comment by FalseMSP** [comment]
I too have this problem, have you found a solution?

**Comment by hkva** [comment]
I'm having trouble reproducing this. Could both of you please let me know what Fabric API version you're using?

**Comment by FalseMSP** [comment]
modlist: 
![image](https://github.com/chunkaligned/fabric-discord-integration/assets/55561022/13ee5665-4bdb-4151-8c87-9675d543af5e)

fabric-api-0.80.0 1.19.4.jar

**Comment by hkva** [comment]
Thanks @FalseMSP and @animator013. I fixed the issue in a9cc4dd and you can download the new release here: https://github.com/chunkaligned/fabric-discord-integration/releases/tag/v1.2.4
